<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleTab</class>
 <widget class="QWidget" name="SingleTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>65</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="savedBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nameBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>16</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>25</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-image:url(:pic2/close2.png);border-style: flat;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
